dt/uni04delta/kustomization.yaml

```diff
@@ -0,0 +1,69 @@
+---
+apiVersion: kustomize.config.k8s.io/v1alpha1
+kind: Component
+
+transformers:
+  - |-
+    apiVersion: builtin
+    kind: NamespaceTransformer
+    metadata:
+      name: _ignored_
+      namespace: openstack
+    setRoleBindingSubjects: none
+    unsetOnly: true
+    fieldSpecs:
+      - path: metadata/name
+        kind: Namespace
+        create: true
+
+components:
+  - ../../lib/networking/metallb
+  - ../../lib/networking/netconfig
+  - ../../lib/networking/nad
+  - ../../lib/control-plane
+
+patches:
+- target:
+    version: v1beta1
+    kind: NetConfig
+    name: netconfig
+  patch: |-
+    - op: add
+      path: /spec/networks/-
+      value:
+        dnsDomain: _replaced_
+        name: StorageMgmt
+        subnets:
+          - _replaced_
+        mtu: 1500
+
+replacements:
+  - source:
+      kind: ConfigMap
+      name: network-values
+      fieldPath: data.storagemgmt.dnsDomain
+    targets:
+    - select:
+        kind: NetConfig
+      fieldPaths:
+      - spec.networks.[name=StorageMgmt].dnsDomain
+
+  - source:
+      kind: ConfigMap
+      name: network-values
+      fieldPath: data.storagemgmt.mtu
+    targets:
+    - select:
+        kind: NetConfig
+      fieldPaths:
+      - spec.networks.[name=StorageMgmt].mtu
```

Comment:
need to fix network names to lower at all the places like https://github.com/openstack-k8s-operators/architecture/pull/171